include/flatbuffers/flatbuffers.h

```diff
@@ -1490,14 +1523,15 @@ class FlatBufferBuilder {
     return CreateString(str, strlen(str));
   }
 
+  // clang-format off
```

Comment:
How:
- `CreateString(flatbuffers::string_view str)`
- `template<typename T> Offset<String> CreateString(const T &str)`

connected with `std::vector` and `std::string`?